members/README.md

```diff
@@ -114,3 +114,26 @@ Creates new request for member introduction
 - **Error Response:**
     - **Code:** 500
         - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+## Patch /members/archiveMembers/:username
```

Comment:
it is recommended to use hyphen-separated words for better readability rather than camel case for `archiveMembers`